src/main/java/org/lsc/beans/syncoptions/PropertiesBasedSyncOptions.java

```diff
@@ -197,6 +197,13 @@ public String getCondition(LscModificationType operation) {
 		return result;
 	}
 
+	public String getPostHookOutputFormat() {
+		if (conf.getHooks() == null || conf.getHooks().getOutputFormat() == null) {
+			return "";
```

Comment:
As said earlier, the result of this method should be an enum.
Also, I would make clearer here which is the default value.